apps/zotonic_mod_admin/src/models/m_admin_status.erl

```diff
@@ -39,8 +40,11 @@
 ]).
 
 -spec m_get( list(), zotonic_model:opt_msg(), z:context()) -> zotonic_model:return().
-m_get([ <<"zotonic_version">> | Rest ], _Msg, _Context) ->
-    {ok, {z_convert:to_binary(?ZOTONIC_VERSION), Rest}};
+m_get([ <<"zotonic_version">> | Rest ], _Msg, Context) ->
+    case z_acl:is_admin(Context) of
```

Comment:
Maybe also let admin users see this? So `orelse z_acl:is_allowed(use, mod_admin, Context)`

Reply:
It did not have a check yet. It looked like it needed one, because this is accessible via the external api, so I added the same check as is done with the db and otp version checks. We could address that in a different issue/pr I think?